lib/plaid.rb

```diff
@@ -33,7 +33,11 @@ module Plaid
   #
   # Returns truey value for relaxed mode and falsey for strict.
   def relaxed_models?
-    defined?(@relaxed_models) && @relaxed_models
+    if defined?(@relaxed_models)
+      @relaxed_models
```

Comment:
I think it would be helpful to set `Plaid.relaxed_models = false` for tests. This would create an incentive to add more fields to the client library.

Reply:
:zap: